src/app-layout/visual-refresh/tools.scss

```diff
@@ -35,10 +34,6 @@ viewport size and state of the Tools drawer.
 
   pointer-events: none;
 
-  @include styles.media-breakpoint-up(styles.$breakpoint-xx-large) {
-    #{custom-props.$toolsWidth}: 360px;
-  }
```

Comment:
Did some historical research, did not find any references on this one, so I am going to remove this along with all other code